### PR TITLE
Test: Add tests for validations on personal detail fields

### LINKED ISSUE #64: Test: Add tests for validations on personal detail fields
## Acceptance Criteria
- [ ] Assert if validation check pops up for username - "Must be between 5-25 characters long. Can only contain alphabets, numbers and underscore _"
- [ ] Asset if validation check pops up for name field - "Must be between 2-30 characters long. Can only contain alphabets, whitespace and dash -"
- [ ] Use the react testing library for writing tests

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by Aaishpra**
### Description

Solves the following

- Assert if validation check pops up for username - "Must be between 5-25 characters long. Can only contain alphabets, numbers and underscore _"

- Asset if validation check pops up for name field - "Must be between 2-30 characters long. Can only contain alphabets, whitespace, and dash -"

- Test to ensure the required fields are not empty for username and name.

Fixes #64

### Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bugfix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

All the tests are passing in VS code terminal

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] I have written Kotlin Docs whenever is applicable

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Aaishpra** [comment]
@mtreacy002 please review this.

**Review by mtreacy002 — DISMISSED** [review]
@Aaishpra , LGTM. Well done for completing the task in such a short time, 👍. Thank you for your contribution to the project.

**Comment by mtreacy002** [comment]
@anitab-org/bridgeintech-maintainers or @anitab-org/coding-team . Can someone also please help reviewing this PR? Thanks in advance.

File: src/tests/PersonalDetails.test.js
Comment (Kajol-Kumari) [inline src/tests/PersonalDetails.test.js]: Please remove the extra spaces from this file and just keep single line space where-ever required.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Aaishpra Minor changes required. Rest Looks good to me :+1:

**Comment by Aaishpra** [comment]
@mtreacy002 @Kajol-Kumari  Please have a look now.

**Review by mtreacy002 — APPROVED** [review]
This one too, LGTM, @Aaishpra 👍 . Thanks again for the fix.

**Review by Kajol-Kumari — APPROVED** [review]
@Aaishpra thanks for making the requested changes. This looks good to me now :)

**Review by keshakaneria — APPROVED** [review]
Squash the commits @Aaishpra and you are good to go! 🎉

**Comment by mtreacy002** [comment]
@Kajol-Kumari and @keshakaneria, thanks for your review. Although, they are not yet counted as approval. Can you please tick the `Approve` box to officially approve the PR? Other than `Approve`, the review doesn't count as being approved (see the second image)
Screen Shot 2020-09-16 at 8 40 21 am
Screen Shot 2020-09-16 at 8 41 09 am

@isabelcosta or @mayburgos or @annabauza , Is there any restrictions for other reviewers on BridgeInTech repositories, such as only maintainers can approve a PR? This seems to be the case since @Techno-Disaster also already gave his approval on PR #93 and the approval also didn't count. Can you please help fix this, since this is blocking the project. Thanks

Screen Shot 2020-09-16 at 8 49 01 am

**Comment by keshakaneria** [comment]
@mtreacy002 I suppose its merged now so no need to do this. Sorry for the late response before merging but will take care next time to check that option too :)

**Comment by mtreacy002** [comment]
> @mtreacy002 I suppose its merged now so no need to do this. Sorry for the late response before merging but will take care next time to check that option too :)

No worries, @keshakaneria, it's just a little hiccup that could happen to anyone. Thanks anyway for giving time to give a review 😉

**Comment by mtreacy002** [comment]
While you're here, can you please give approval on PR #93? Just want to confirm that someone who is not a maintainer can approve the PR. Thanks heaps for helping.

**Comment by mtreacy002** [comment]
Sorry, cancel that. It's also been approved and merged (PR #93) 😁

**Pull request merged**
